<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EXEEditor</class>
 <widget class="QWidget" name="EXEEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="uiStr1Tab">
      <attribute name="title">
       <string>UI Strings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="uiStr1Tbl"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiStr2Tab">
      <attribute name="title">
       <string>More UI Strings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="uiStr2Tbl"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="itemStrTab">
      <attribute name="title">
       <string>Item Strings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="itemStrTbl"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="magicStrTab">
      <attribute name="title">
       <string>Magic Strings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="magicStrTbl"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shopTab">
      <attribute name="title">
       <string>Shops</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QComboBox" name="shopCombo"/>
       </item>
       <item>
        <widget class="QTableView" name="shopTbl"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
